webapp/channels/src/components/widgets/advanced_textbox/advanced_textbox.tsx

```diff
@@ -124,26 +126,28 @@ const AdvancedTextbox = ({
                 />)
             }
 
-            {/* Character count display */}
-            {showCharacterCount && (isTooLong || isTooShort || internalError) && (
-                <div
-                    className={classNames('AdvancedTextbox__character-count', {
-                        'exceeds-limit': isTooLong,
-                        'below-minimum': isTooShort,
-                    })}
-                >
-                    {value.length}{'/'}
-                    {isTooShort ? minLength : maxLength}
-                </div>
-            )}
-
-            {/* Error message display */}
-            {internalError && (
-                <div className='AdvancedTextbox__error-message'>
-                    <i className='icon icon-alert-circle-outline'/>
-                    <span>{internalError}</span>
-                </div>
-            )}
+            <div className='AdvancedTextbox__error-wrapper'>
+                {/* Error message display */}
+                {internalError && (
+                    <div className='AdvancedTextbox__error-message'>
+                        <i className='icon icon-alert-circle-outline'/>
+                        <span>{internalError}</span>
+                    </div>
+                )}
+
+                {/* Character count display */}
+                {showCharacterCount && (isTooLong || isTooShort || internalError) && (
+                    <div
+                        className={classNames('AdvancedTextbox__character-count', {
+                            'exceeds-limit': isTooLong,
+                            'below-minimum': isTooShort,
+                        })}
+                    >
+                        {value.length}{'/'}
+                        {isTooShort ? minLength : maxLength}
+                    </div>
+                )}
+            </div>
```

Comment:
Tests for this?

Reply:
I'm adding all tests today - unit and integration.

Reply:
They're added by Pablo now for this component in base branch.